src/component/contents/DraftEditorLeaf.react.js

```diff
@@ -145,6 +148,13 @@ class DraftEditorLeaf extends React.Component {
       return Object.assign(map, style, mergedStyles);
     }, {});
 
+    if (customStyleFn) {
+      const newStyles = customStyleFn(styleSet);
+      if (newStyles !== undefined) {
```

Comment:
Any reason to restrict this check to `undefined`?

Relatedly, could this perform the `Object.assign` with `null` or `undefined` as supplied by the style function, since a no-op should be fine?